push/src/main/AndroidManifest.xml

```diff
@@ -3,90 +3,104 @@
-            android:exported="true"
-            android:process=":pushservice" />
-        <service
-            android:name="com.xiaomi.push.service.PushServiceMain"
+        <activity
+            android:name=".RemoveDozeActivity"
+            android:exported="true" />
+
+
+        <activity
+            android:name=".ManageSpaceActivity"
             android:exported="true"
-            android:process=":pushservice"
-            tools:ignore="ExportedService" />
+            android:label="@string/title_activity_advance_setting" />
 
         <!--
         We use custom push service instead
         <service android:name="com.xiaomi.push.service.XMPushService"
             android:exported="true"
             android:process=":pushservice"/>
         -->
-        <service
-            android:name=".push.service.notificationcollection.NotificationListener"
-            android:permission="android.permission.BIND_NOTIFICATION_LISTENER_SERVICE">
+        <activity
+            android:name="top.trumeet.mipushframework.wizard.WelcomeActivity"
+            android:icon="@mipmap/ic_launcher"
+            android:theme="@style/AppTheme.Wizard">
             <intent-filter>
-                <action android:name="android.service.notification.NotificationListenerService" />
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-        </service>
+        </activity>
+
+        <activity
+            android:name="top.trumeet.mipushframework.wizard.CheckRunInBackgroundActivity"
+            android:theme="@style/AppTheme.Wizard" />
+        <activity
+            android:name="top.trumeet.mipushframework.wizard.FinishWizardActivity"
+            android:theme="@style/AppTheme.Wizard" />
+
+        <activity
+            android:name="top.trumeet.mipushframework.wizard.UsageStatsPermissionActivity"
+            android:theme="@style/AppTheme.Wizard" />
+        <activity
+            android:name="top.trumeet.mipushframework.settings.MainActivity"
+            android:configChanges="screenLayout|screenSize|orientation"
+            android:label="@string/preference_title" />
+        <activity
+            android:name="top.trumeet.mipushframework.permissions.ManagePermissionsActivity"
+            android:configChanges="screenLayout|screenSize|orientation"
+            android:label="@string/action_edit_permission" />
+
+        <activity
+            android:name="top.trumeet.mipushframework.event.RecentActivityActivity"
+            android:label="@string/recent_activity_title" />
+        <activity
+            android:name="top.trumeet.mipushframework.help.HelpActivity"
+            android:label="@string/helplib_title"
+            android:theme="@style/HelpTheme.DayNight" />
+
+        <meta-data
+            android:name="xposedmodule"
+            android:value="true" />
```

Comment:
According to https://github.com/MiPushFramework/MiPushFramework/commit/9a1174c155281423e10e75c10d9d59e5fe84aacd , there should be no more xposed module related content in Push. At the same time, the xposed function in push has been moved to a new independent software [MiPushEnhancement](https://github.com/MiPushFramework/MiPushEnhancement), and these residual codes should be deleted.

Reference: https://t.me/MiPushFramework/12